src/armado/compressed_index.py

```diff
@@ -220,9 +269,11 @@ def similar_terms(self, t):
         # It's quadratic on the length of the term,
         # so we'll limit the length of the subterm to 20 characters max
         #
-        for tv in iter( t[a:a+l]
-                        for l in xrange(min(20,len(t)),0,-1)
-                        for a in xrange(len(t)-l+1) ):
+        for tv in iter(
+            t[a:a + l]
+            for l in xrange(min(20, len(t)), 0, -1)
+            for a in xrange(len(t) - l + 1)
+        ):
```

Comment:
acá queda mucho más claro si todo ese `iter(...)` se arma en una linea anterior, y el `for` usa eso...